manifests/init.pp

```diff
@@ -7,6 +7,9 @@
 #
 # $ensure_packages_version::    control extra packages version, it's passed to ensure parameter of package resource
 #
+# $manage_service:                     control the service, wether it should be started / enabled or not. usefull, if the
```

Comment:
This should be an advanced parameter, under the `Advanced Parameters`

Reply:
hi @ekohl ,
moved to Advanced Parameters :)